src/components/form/form.style.js

```diff
@@ -100,4 +103,32 @@ const StyledForm = styled.form`
   `}  
 `;
 
+StyledForm.defaultProps = {
+  theme: baseTheme
+};
+
+StyledForm.propTypes = {
+  theme: PropTypes.object,
+  stickyFooter: PropTypes.bool,
+  fixedBottom: PropTypes.bool
+};
+
+StyledFormFooter.defaultProps = {
+  theme: baseTheme
+};
+
+StyledFormFooter.propTypes = {
```

Comment:
It looks like the `StyledFormFooter` StyledComponent also supports these props: `stickyFooter`, `buttonAlign`. Should we add `propTypes` here for them?

Reply:
Yep good spot I've corrected that now

Reply:
Thanks :white_check_mark: